packages/eth-explorer/src/AccountPage.js

```diff
@@ -10,26 +10,47 @@ import AccountBalance from './AccountBalance'
 import AccountInfo from './AccountInfo'
 import AccountTransactions from './AccountTransactions'
 
+let currentNetId = null
```

Comment:
it will be better to use the static property of AccountPage instead of the external variable